sdcm/utils/loader_utils.py

```diff
@@ -63,21 +63,26 @@ def _create_counter_table(self):
                     AND speculative_retry = '99.0PERCENTILE';
             """)
 
-    def assemble_and_run_all_stress_cmd(self, stress_queue, stress_cmd, keyspace_num):
+    def assemble_and_run_all_stress_cmd(self, stress_queue, stress_cmd, keyspace_num, ks_names=None):
```

Comment:
This logic gets complicated and there's no clear doc what takes precedence over what.
Consider splitting this function.

Reply:
Agree about getting more complex. I will consider splitting.